android/app/src/main/assets/capacitor.plugins.json

```diff
@@ -2,21 +2,5 @@
 	{
 		"pkg": "@capacitor-community/capacitor-googlemaps-native",
 		"classpath": "com.hemangkumar.capacitorgooglemaps.CapacitorGoogleMaps"
-	},
-	{
-		"pkg": "@capacitor/app",
```

Comment:
Not sure why this is removed after `npx cap sync` either..

Reply:
because you removed @capacitor/app from the package.json